stubs/tensorflow/tensorflow/keras/layers/__init__.pyi

```diff
@@ -442,4 +442,21 @@ class GaussianDropout(Layer[tf.Tensor, tf.Tensor]):
         name: str | None = None,
     ) -> None: ...
 
+class MaxPooling2D(Layer[tf.Tensor, tf.Tensor]):
+    def __init__(
+        self,
+        pool_size: int | tuple[int, int] = (2, 2),
+        strides: int | tuple[int, int] | None = None,
+        padding: Literal["valid", "same"] = "valid",
+        data_format: None | Literal["channels_last", "channels_first"] = None,
+        name: str | None = None,
+    ) -> None: ...
+
+MaxPool2D = MaxPooling2D
+
+class GlobalAveragePooling2D(Layer[tf.Tensor, tf.Tensor]):
```

Comment:
Similar comments as above re: ordering of `| None`, and the base class here is `BaseGlobalPooling`